packages/client/containers/Signout/signout.ts

```diff
@@ -6,16 +6,15 @@ import {SegmentClientEventEnum} from '../../types/graphql'
 const signout = (atmosphere: Atmosphere, history: any) => {
   window.localStorage.removeItem(APP_TOKEN_KEY)
   SendClientSegmentEventMutation(atmosphere, SegmentClientEventEnum.UserLogout)
-  /* reset the app state, but preserve any pending notifications: */
-  if (history) {
-    history.replace('/')
-  }
   atmosphere.eventEmitter.emit('addSnackbar', {
     key: 'logOut',
     message: 'You’ve been logged out successfully.',
     autoDismiss: 5
   })
-  atmosphere.close()
+  setTimeout(() => {
```

Comment:
Interesting, why do this async?

Reply:
we weren't properly killing the socket connection on logout!